docs/modules/ROOT/pages/api/account.adoc

```diff
@@ -259,3 +259,47 @@ Emitted when a `public_key` is added.
 ==== `[.contract-item-name]#++OwnerRemoved++#++(removed_owner_guid: felt252)++` [.item-kind]#event#
 
 Emitted when a `public_key` is removed.
+
+== Presets
+
+[.contract]
+[[Account]]
+=== `++Account++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/release-v0.8.0-beta.0/src/presets/account.cairo[{github-icon},role=heading-link]
+
+```javascript
+use openzeppelin::presets::Account;
+```
+
+Basic Account preset leveraging xref:#AccountComponent[AccountComponent].
```

Comment:
```suggestion
Basic account contract leveraging xref:#AccountComponent[AccountComponent].
```

we never define "preset" in the docs, don't we? I think this might help users understand. also, "account" [should be lowercase](https://github.com/OpenZeppelin/cairo-contracts/issues/726#issuecomment-1714082530) since it's not a module, we're using the word to describe the module